engine/src/test/java/org/hibernate/validator/test/internal/engine/valuehandling/OptionalWithPrivateGetterTest.java

```diff
@@ -41,7 +42,7 @@ public void testConstraintCanBePlacedOnPrivateGetter() {
 	public void testConstraintOnPrivateGetterReturnsConstraintViolation() {
 		Validator validator = getValidator();
 
-		LocalDate[][] testData = { { LocalDate.MAX, LocalDate.MIN }, { LocalDate.now(), null }, { null, null } };
+		LocalDate[][] testData = { { LocalDate.MAX, LocalDate.MIN }, { LocalDate.now( ZoneId.systemDefault() ), null }, { null, null } };
```

Comment:
@gsmet  +1 
I'm not very happy about them as well. That's why I've done them all in a separate commit so it can be removed if we decide so. These things comes from those `jdk-unsafe`, `jdk-deprecated`, `jdk-non-portable`. They seems to be useful to have in general (judging  by the names).